### ISSUE TITLE
Unity8 Apps?

### ISSUE THREAD

**Issue by star-buck** [yunit-io/yunit#37]
Do Unity8 code repositories include any (core) applications, such as filemanager, terminal, imageviewer, etc.? Any list or overview?

Would be great to preserve and make them run on wayland, so they could be used under Yunit, Plasma/Mobile or any other Wayland-compatible DE/device.

**Comment by Pharaone** [comment]
Nope, unity8 official repository doesn't include any core application

**Comment by star-buck** [comment]
Is the code for those apps anywhere else or are they closed source?

**Comment by Pharaone** [comment]
Those apps are all on launchpad.net

**Comment by Pharaone** [comment]
Here https://launchpad.net/ubuntu-system-apps

**Comment by star-buck** [comment]
Thanks! How about adding the code here (on yunit-io), too?

**Comment by jsalatas** [comment]
We need to fork other repos as well beside the unity 8 repo that is already forked. I guess it will take some time until we figure out what is needed :)

I'm currently working on identifying yunit (former unity 8) dependencies that need forking. I need to have all libraries/components down to mir client/server. Hopefully I'll have an update for that soon.

Can someone take care any other components that need merging? For every cmponent/app we need to convert from bazaar to git as described at https://design.canonical.com/2015/01/converting-projects-between-git-and-bazaar/ and then push to a repo here. If you do convert make sure to push all tags (or branches) so that we can keep the history....

**Comment by jsalatas** [comment]
I have created two projects for this 

https://github.com/yunit-io/yunit/projects/1
https://github.com/yunit-io/yunit/projects/2

first is assigned to me but any help is more than welcomed :)
if someone is interested to take project two, please let me know

**Comment by Pharaone** [comment]
I think  I have some other repo that need forking: https://launchpad.net/ubuntu-terminal-app , https://launchpad.net/qtubuntu and some others

**Comment by star-buck** [comment]
What's qtubuntu for??

Not entirely sure, but imo it should be a high priority goal to get rid of
(like mir) and re-base on native qt instead, especially if this
qtubuntu/qpa layer is only needed for enabling mir in the first place:
http://bazaar.launchpad.net/~phablet-team/qtubuntu/trunk/view/head:/README

**Comment by labsin** [comment]
Qtubuntu is the  qpa layer like qtwayland but for Ubuntu/mir. It also got helper functions for the Ubuntu touch life cycle management.

**Comment by dfiloni** [comment]
@jsalatas : you can take a look at debian/control file for project dependencies (Build-Depends and Depends sections), this should help you to find required apps/libraries/components faster ;)
packages.ubuntu.com can help you to find them too, for example take a look at:
http://packages.ubuntu.com/zesty/unity8-desktop-session

**Comment by kicsyromy** [comment]
@jsalatas Stupid question... What is the difference between https://github.com/yunit-io/yunit/projects/1 and https://github.com/yunit-io/yunit/projects/2? And where would https://launchpad.net/ubuntu-ui-toolkit fit in?

**Comment by jsalatas** [comment]
@kicsyromy it is just dependencies (project 1) vs applications (project 2). yunit cannot be built without dependencies but it can be built without apps. ;)

**Comment by kicsyromy** [comment]
@jsalatas Where would runtime dependencies fit, in this case? Plugins and such...

**Comment by rolandixor** [comment]
1

**Comment by Leppan** [comment]
I'd imagine plugins would act like apps; they could simply be installed and uninstalled. They just always run in the background.
So they'd be in the app section.

**Comment by Leppan** [comment]
Ive just forked the notes app.

**Comment by Goddard** [comment]
What library is init-tool-merge?  I think it is for translations. I was trying to build the ubuntu-terminal-app and got an error.

**Comment by MggMuggins** [comment]
@labsin - Sorry for referencing something so far up, but since Yunit is more focused on desktop users (this is correct, right?), and since the point of qtubuntu is mostly touch stuff, shouldn't removing it be priority, migrating back to qtwayland?
Sorry for most likey useless noise...
